oaknational/Oak-Web-Application
# CTC-60 Create MediaClipInfo component

## Description

Created `MediaClipInfo` component

## Issue(s)

Fixes [CTC-60](https://www.notion.so/oaknationalacademy/OWA-MediaClipInfo-Component-13826cc4e1b1804e958eee40a8f9005e)

## How to test

1. Go to https://deploy-preview-3032--owa-storybook.netlify.thenational.academy/?path=/docs/components-teachercomponents-lessonmediaclipinfo--docs
2. Check ,MediaClipInfo` component in storybook

## Designs
https://www.figma.com/design/iTimVTBZ3XcszhRa9KtD7r/Cycle-2-requirements---Teachers?node-id=227-40795&node-type=frame&m=dev

## Checklist
